package.json

```diff
@@ -71,6 +72,8 @@
     "x-xss-protection": "^1.0.0"
   },
   "dependencies": {
+    "@types/node": "~8.0.27",
+    "@types/pino": "~4.7.0",
```

Comment:
would it fail we don't do this?

We typically depends with `^` in this project.